pages/experiences/filecoin-settings.js

```diff
@@ -52,13 +54,9 @@ const EXAMPLE_CODE = `import * as React from "react";
 import { FilecoinSettings } from "slate-react-system";
 import { createPow } from "@textile/powergate-client";
 
-const PowerGate = createPow({ host: "http://pow.slate.textile.io:6002" });
+const PowerGate = createPow({ host: "${POWERGATE_HOST}" });
```

Comment:
People don't have access to `POWERGATE_HOST` outside of Slate, so please change this back.